docs/overlay.md

```diff
@@ -27,6 +27,14 @@ title: Overlay
 >
   <Text>Hello from Overlay!</Text>
 </Overlay>;
+
+
+<Overlay
+  isVisible={this.state.isVisible}
+  onBackdropPress={this.onBackdropPress}
```

Comment:
Can add here `onBackdropPress={() => this.setState({ isVisible: false })}`. Just so they know how to close it

Reply:
sure !!

Reply:
@iRoachie I have added the changes you required. I amended the previous commit